governance/contributor-ladder.md

```diff
@@ -0,0 +1,112 @@
+Actions to help this include engaging in discussions, providing feedback on proposals, reviewing code contributions, and offering assistance in any way that aligns with your skills and expertise.
+By actively participating and contributing, you demonstrate your commitment to the community and showcase the values that the Environmental Sustainability TAG holds dear.
+
+* Helping out and mentor new contributors in their open source journey.
+* Helping out community members and leaders if they need support.
+* Demonstrating empathy and embody the community values.
+
+## Leading a Working Group
+
+Since leadership roles naturally change over time, it is worth your while to take a leadership position in an established working group.
+Within the TAG for Environmental Sustainability, dedicated working groups carry out several ongoing initiatives.
+However, if the current working groups do not provide space for the topic you are interested in, you could start a new working group.
+A working group must have community support and requires that you find sufficient backing before a working group is formed.
+To start this process, create a GitHub issue describing the initial charter in which you outline the motivation, scope, goals, and deliverables of the proposed working group.
+This working group charter document should form the basis for discussions with the TAG before initiating an official vote by the TAG co-chairs, which also requires approval by the CNCF TOC.
+It is important to present the working group proposal during TAG meetings and be vocal about the discussions, actively solicit feedback and input, and gain support from other members.
+
+* Taking on more responsibility and ownership within TAG.
+* Leading a community group within TAG.
+* Conflict resolution.
+
+## Nominated as Tech Lead
+
+To serve in the role as Tech Lead, you have to go through the [tech lead proposal process](https://github.com/cncf/tag-env-sustainability/blob/main/governance/tech-lead-proposal-process.md).
+Assuming the role of a technical lead within the Environmental Sustainability TAG places greater responsibilities on your shoulders as part of the leadership team.
+This roles provides you with the opportunity to actively support the leadership team by taking charge of a specific topic within the TAG.
+As an elected Tech Lead and a member of the leadership team, you will have the chance to delve into the broader aspects of leading the community group, gaining insights into the organizational structures within the CNCF. 
+Additionally, you will play a crucial role in facilitating discussions and resolving any issues that may arise between contributors, ensuring a harmonious and productive community environment.
+Your position as a technical lead allows you to contribute significantly to the overall success and cohesion of the Environmental Sustainability TAG.
+
+* Build partnerships between TAG and projects or other organizations.
```

Comment:
This is "expectations of a tech lead?"

Reply:
Removed it, @catblade do you may have a suggestion what we could add here? We just have one bullet point.

Reply:
Good point.  Can we move some of the content from the above paragraphs into the bullet points?

Like:
* Facilitating discussions
* Chop wood/carry water (or whatever phrase that Max likes)
* Lead WGs and make sure deliverables stay on schedule